src/gatling/scala/uk/gov/hmcts/reform/cmc/performance/processes/Reason.scala

```diff
@@ -0,0 +1,26 @@
+package uk.gov.hmcts.reform.cmc.performance.processes
+
+import io.gatling.core.Predef._
+import io.gatling.core.structure.ChainBuilder
+import io.gatling.http.Predef._
+import uk.gov.hmcts.reform.cmc.performance.simulations.checks.CsrfCheck.{csrfParameter, csrfTemplate}
+import uk.gov.hmcts.reform.cmc.performance.simulations.checks.CurrentPageCheck.currentPageTemplate
+import uk.gov.hmcts.reform.cmc.performance.simulations.checks.{CsrfCheck, CurrentPageCheck}
+
+object Reason {
+
+  def run(implicit postHeaders: Map[String, String]): ChainBuilder = {
+    exec(http("Reason - GET")
+      .get("/claim/reason")
+        .check(CsrfCheck.save)
+        .check(CurrentPageCheck.save))
```

Comment:
Would be nice to not have to do this explicitly for each page.. We need more Scala hacking skills :) Not strong on fixing this now.